src/styles/utils/mixins.scss

```
@@ -0,0 +1,34 @@
+@mixin on-phone {
+  @media (min-width: 488px) {
+    @content;
+  }
+}
+
+@mixin on-tablet {
+  @media (min-width: 768px) {
+    @content;
+  }
+}
+
+@mixin on-desktop {
+  @media (min-width: 1024px) {
+    @content;
+  }
+}
+
+
+@mixin number-of-grid-template-columns {
```

what is a `number of grid`? We can make it short a bit
```suggestion
@mixin grid {
```